### PR TITLE
Fix/dragen germline tool

### PR DESCRIPTION
**Pull request by skanwal**
Fixes https://github.com/umccr/cwl-ica/issues/316

### CODE REVIEW

**Comment by alexiswl** [comment]
Did you also wish to sync the dragen-germline-pipeline and dragen-somatic-with-germline-pipeline as well?

**Comment by alexiswl** [comment]
> Did you also wish to sync the dragen-germline-pipeline and dragen-somatic-with-germline-pipeline as well?

Actually maybe wait for #318 to complete first

File: tools/dragen-germline/4.0.3/dragen-germline__4.0.3.cwl [hunk 1]
```diff
@@ -73,7 +73,7 @@ requirements:
             --file "$(inputs.reference_tar.path)"
           
           # Confirm either of fastq_list, fastq_list_rows or bam_input is defined
-          if [[ "$(boolean_to_int(is_not_null(inputs.fastq_list)) + boolean_to_int(is_not_null(inputs.fastq_list_rows)) + boolean_to_int(is_not_null(inputs.bam_input)))" == "1" ]]; then
+          if [[ "$(boolean_to_int(is_not_null(inputs.fastq_list)) + boolean_to_int(is_not_null(inputs.fastq_list_rows)) + boolean_to_int(is_not_null(inputs.bam_input)))" -ne "1" ]]; then
             echo "Please set no more than one of fastq_list, fastq_list_rows and bam_input for normal sample" 1>&2
```
Comment (alexiswl) [inline tools/dragen-germline/4.0.3/dragen-germline__4.0.3.cwl:77]: Should be _echo "Please set one AND only one of ..._

**Review by alexiswl — CHANGES_REQUESTED** [review]

**Comment by alexiswl** [comment]
Converted to draft as I need to re-push to #318 due to a syntax error in the somatic pipeline.

**Comment by skanwal** [comment]
@alexiswl - updated in https://github.com/umccr/cwl-ica/pull/319/commits/c0b9174e2b3a885825c9459ba4022f0b9a97045e

**Review by alexiswl — APPROVED** [review]

**Pull request merged**